### DISCUSSION TITLE
Opening music panel should close player panel and vice versa

### DISCUSSION THREAD

**Discussion by OwenCMYK** [ppy/osu#27248] [Ideas] [answered]
![image](https://github.com/ppy/osu/assets/42657486/f3fc581e-9cc9-4fa5-8b63-4b8b4b0a5819)

This is pretty self explanatory, but it's not really a big deal. I just think these two panels overlapping looks weird and so they probably shouldn't be able to. Or from a UX standpoint, it requires you to close the music panel before the player panel becomes (reasonably) usable, so it adds an extra click when the player wants to visit their profile after interacting with the music panel.

**Comment by Joehuu** [comment, marked as answer]
Already tracked as an issue in https://github.com/ppy/osu/issues/4726.
